src/AggregationFunction/FunctionSum.php

```diff
@@ -53,6 +59,16 @@ public function processDataSource($data_source)
 				->fetch()
 				->sum;
 		}
+		if ($data_source instanceof \Doctrine\ORM\QueryBuilder) {
+			$data_source_clone = clone $data_source;
```

Comment:
When I used only original QB, I had problems with select part - so I only created clone of QB. I sent something which might be better.